docs/user-guide/migration/pandas.md

```diff
@@ -174,7 +176,7 @@ the values in column `a` based on a condition. When the value in column `c` is e
 In `Pandas` this would be:
 
 ```python
-df.loc[df["c"] == 2, "a"] = df.loc[df["c"] == 2, "b"]
+df.assign(a=lambda df_: np.where(df_.c == 2, df_.b, df_.a))
```

Comment:
slightly more idiomatic would be to use `pandas.Series.where` directly
```python
df.assign(a=lambda df_: df_.a.where(df_.c != 2, df_.b))
```

Reply:
Thanks @MarcoGorelli, I wasn't aware that `Pandas` dataframe includes its own `where` method.